templates/taxi/car_confirm_delete.html

```diff
@@ -0,0 +1,16 @@
+{% extends "base.html" %}
+{% load crispy_forms_filters %}
+
+{% block content %}
+  <h2>Delete car</h2>
+  <p>
+    Are you sure you want to delete car
+    {{ car }} ?
+  </p>
+  <form method="post" action="">
+    {% csrf_token %}
+    {{ form|crispy }}
+    <button type="submit" class="btn btn-danger">Delete</button>
```

Comment:
Also, you need to add one more button according to check_list paragraph [#5](https://github.com/mate-academy/py-taxi-service-forms/blob/master/checklist.md)

Reply:
Hmm, I, somehow missed that paragraph, thanks